From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@PatrickAlexander__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@PatrickAlexander 
Thank you for your contribution! I'm sending this to the author for validation.
--Carolyn

cjgronlund commented:
@stevestein
Please check with PR and let me know whether to accept it.

stevestein commented:
@PatrickAlexander 
Thanks for pointing this out. I'm incorporating your suggestion of required permissions into each of the 'Copy SQL Database' articles. The updated versions should appear online by the end of the week.

cjgronlund commented:
@PatrickAlexander 
Thank you again for contributing. I'm closing this PR now.

kgremban committed:
Updated X.509 supported certificate types

Commit:
include more clear instructions for event hubs sas token

zhiweiwangmsft committed:
Add alert catalog

Commit:
initial catalog

Commit:
update format

Commit:
finish sync

Commit:
format

Commit:
backslash

Commit:
adjust

Commit:
add adds

Commit:
ADDS 2

Commit:
format

Commit:
Style update

Commit:
adfs 1

Commit:
adfs 2

Commit:
ADFS 3

mumian committed:
[HDI] remove an article

anrothMSFT committed:
typo on quota values

mumian committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into 0319-dj-omid

mumian committed:
update

mumian committed:
[HDI] remove an article

Commit:
format udpate

Commit:
minor

Commit:
minor

mumian committed:
update

mumian committed:
revert links

mumian committed:
update links

Commit:
minor style

Commit:
Style update

Commit:
update format

update format

Commit:
format

cynthn committed:
Update maintenance-and-updates.md

cynthn committed:
Update maintenance-and-updates.md

AjayKumar-N committed:
(ForumsTeam) Broken link needs to fixed.

In the referenced document https://docs.microsoft.com/en-us/azure/service-bus-relay/relay-faq
The hyperlink see "Azure Support FAQs." is broken and needs to be fixed using this url: https://azure.microsoft.com/en-in/support/faq/

Commit:
format again

cherylmc committed:
metadata

Commit:
format again

cherylmc committed:
metadata

cherylmc committed:
metadata

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into includes

charwen committed:
Added PPS and CPS

cherylmc committed:
metadata

cherylmc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into includes

Commit:
review1

rakshitatandon committed:
Update service-fabric-windows-cluster-windows-security.md

Update gMSA details

rakshitatandon committed:
Update service-fabric-windows-cluster-windows-security.md

rakshitatandon committed:
Update service-fabric-windows-cluster-windows-security.md

rakshitatandon committed:
Update service-fabric-windows-cluster-windows-security.md

rakshitatandon committed:
Update service-fabric-windows-cluster-windows-security.md

KumudD committed:
draft added

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into qslbportal0322

KumudD committed:
tweak

KumudD committed:
tweak

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into qslbportal0322

KumudD committed:
updated frontend info

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into qslbportal0322

mimig1 committed:
Cosmos DB: Added anchors and modified layout of next steps

mimig1 committed:
Cosmos DB: Removed redirect file from documentdb folder and added to redirect file.

KumudD committed:
addressed PR Reviewer edits

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into qslbportal0322

KumudD committed:
fixed pool info

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into qslbportal0322

douglaslMS committed:
Finished fixing metadata in last ADF article.

curtand committed:
[Azure AD LinkedIn integration] correct group policy template name

KumudD committed:
tweak

douglaslMS committed:
Added link to tutorial for more info about VSTS.

PRMerger10 committed:
Merge pull request #36031 from douglaslMS/adf-finish-metadata

Finished fixing metadata in last ADF article.

rothja committed:
Per-minute to per-second pricing fixes MicrosoftDocs/azure-docs#5167

PRMerger13 committed:
Merge pull request #36032 from curtand/linkedin0322

[Azure AD LinkedIn integration] correct group policy template name

mumian committed:
fix the redirect links

cherylmc committed:
metadata

PRMerger14 committed:
Merge pull request #36033 from douglaslMS/adf-vsts-link

Added link to tutorial for more info about VSTS.

ktoliver committed:
Merge pull request #36017 from KumudD/qslbportal0322

Create a Basic Load Balancer - portal

PRMerger15 committed:
Merge pull request #35938 from cynthn/patch-71

Update maintenance-and-updates.md - win

v-shils committed:
Merge pull request #36025 from mimig1/patch-229

Cosmos DB: Added anchors and modified layout of next steps

v-shils committed:
Merge pull request #36028 from mimig1/03222018-scrub

Cosmos DB: Removed redirect file from documentdb folder and added to …

Commit:
Updated script for TLS1.2

KumudD committed:
removed preview text

v-shils committed:
Corrected typo

v-shils committed:
Corrected typo

Commit:
Merge pull request #36036 from jeffgilb/322

Updated script for TLS1.2

klaaslanghout committed:
update regions we support

Commit:
March Release

ktoliver committed:
Merge pull request #35984 from rakshitatandon/master

Update gMSA details for windows security

ktoliver committed:
Merge pull request #36037 from cherylmc/exinclude

metadata

ktoliver committed:
Merge pull request #36038 from KumudD/removelbpreviewtext

removed SLB preview text

Brenduns committed:
revision per Tech Review

PRMerger10 committed:
Merge pull request #36040 from fimguy/connectors322

March Release

douglaslMS committed:
Made suggested SEO changes for 2 SSIS-ADF tutorials.

Commit:
Minor updates

v-shils committed:
Merge pull request #35962 from cherylmc/includes

Includes

charwen committed:
Added tags

v-shils committed:
Merge pull request #35653 from mumian/0319-dj-omid

[HDI] remove an article

charwen committed:
Updated the tags

PRMerger17 committed:
Merge pull request #36043 from Brenduns/relnote-1711-march-22

revision per Tech Review

charwen committed:
updated the header

charwen committed:
Updated the table

tamram committed:
adding section on how to retrieve connection string

Commit:
Merge pull request #35937 from cynthn/patch-70

Update maintenance-and-updates.md

Commit:
fixed typo

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#5702

Commit:
add table for ubuntu dependencies

douglaslMS committed:
Added info about v2 role requirements per looong email thread.

douglaslMS committed:
Adding required TOC updates.

tamram committed:
adding image

wesmc7777 committed:
user story 1087924

GitHubber17 committed:
Merge pull request #35678 from mumian/0319-spark-connector

[HDI] remove an article

GitHubber17 committed:
Merge pull request #35482 from zhiweiwangmsft/patch-19

Add CH alert catalog

mattbriggs committed:
update topic

GitHubber17 committed:
Merge pull request #35970 from charwen/patch-48

Added PPS and CPS

GitHubber17 committed:
Merge pull request #36034 from rothja/persecondpricing

Per-minute to per-second pricing fixes MicrosoftDocs/azure-docs#5167

MikeRayMSFT committed:
Add note about pricing

GitHubber17 committed:
Merge pull request #36039 from klaaslanghout/patch-19

update regions we support

PRMerger14 committed:
Merge pull request #36053 from douglaslMS/adf-roles

Added info about v2 role requirements per looong email thread.

MikeRayMSFT committed:
Remove some hyphens.

Commit:
March Release

wesmc7777 committed:
relocating this faq

GitHubber17 committed:
Merge pull request #36045 from jeffgilb/322.1

Minor updates

barbkess committed:
fix typo - fixes MicrosoftDocs/azure-docs/issues/6141

PRMerger18 committed:
Merge pull request #36057 from mattbriggs/mdb_20180322_ps_quick

Remove unused keyvault step

ajlam committed:
Merge pull request #1 from MicrosoftDocs/master

Update fork

Commit:
Fixed troubleshooting articles

MikeRayMSFT committed:
Fix broken link.

MikeRayMSFT committed:
Add acronym

PRMerger20 committed:
Merge pull request #36054 from suhuruli/master

Fix java tutorial and linux set up

PRMerger20 committed:
Merge pull request #36060 from fimguy/connectors322

March Release V2 Update

Commit:
Merge pull request #36055 from wesmc7777/redis_updates

user story 1087924

Ja-Dunn committed:
Merge pull request #36046 from tamram/tamram-0322a

adding section on how to retrieve connection string

Commit:
Merge pull request #36063 from jeffgilb/322.2

Fixed troubleshooting articles

PRMerger12 committed:
Merge pull request #36061 from barbkess/git-issues

fix typo - fixes MicrosoftDocs/azure-docs/issues/6141

ajlam committed:
Update PG and MySQL portal screenshots

ktoliver committed:
Merge pull request #36044 from douglaslMS/adf-tutorial-seo

Made suggested SEO changes for 2 SSIS-ADF tutorials.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

tfitzmac committed:
changed description of filtering

ktoliver committed:
Merge pull request #36067 from ajlam/master

Update PG and MySQL portal screenshots

v-alje committed:
Merge pull request #36070 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger18 committed:
Merge pull request #36058 from MikeRayMSFT/20180322_SQLLicensing

Add note about pricing

PRMerger18 committed:
Merge pull request #36049 from tfitzmac/0322egsubject

clarify subject for custom topics

PRMerger20 committed:
Merge pull request #35669 from anrothMSFT/patch-1

typo on quota values

Commit:
Merge pull request #35943 from AjayKumar-N/patch-24

(ForumsTeam) Broken link needs to fixed.

Commit:
Merge pull request #33239 from kgremban/feb12-x509

Updated X.509 supported certificate types

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
